be2-scala/src/main/scala/ch/epfl/pop/model/network/MethodType.scala

```diff
@@ -15,6 +15,7 @@ object MethodType extends Enumeration {
   val HEARTBEAT: Value = MatchingValue("heartbeat")
   val GET_MESSAGES_BY_ID: Value = MatchingValue("get_messages_by_id")
 
+  val GREET_SERVER: Value = MatchingValue("greet_server")
```

Comment:
Absolute nitpick: i think we shoud regroup the MethodType together, the line return don't serve much